lib/Sema/TypeCheckNameLookup.cpp

```diff
@@ -555,13 +555,21 @@ void TypeChecker::performTypoCorrection(DeclContext *DC, DeclRefKind refKind,
 }
 
 static InFlightDiagnostic
-diagnoseTypoCorrection(TypeChecker &tc, DeclNameLoc loc, ValueDecl *decl) {
+diagnoseTypoCorrection(TypeChecker &tc,
+                       DeclNameLoc loc, 
```

Comment:
You don't necessarily have to fold the entire signature over, just those that go over the line-limit.  If it makes it easier, you can run `clang-format` with the default LLVM style to have it wrap these for you.